feature-libs/product-configurator/rulebased/cpq/occ/cpq-configurator-occ.service.ts

```diff
@@ -121,6 +121,26 @@ export class CpqConfiguratorOccService {
     );
   }
 
+  getConfigIdForQuoteEntry(
```

Comment:
Please, add a comment to the method

Reply:
@ChristophHi, could you have a look?

Reply:
@ChristophHi, `getConfigIdForQuoteEntry` method does not exist anymore.
This method will be mocked in `cpq-configurator-rest.adapter.spec.ts` test class (see line 102)

Reply:
Done!